src/constants/shutdowns.json

```diff
@@ -166,153 +166,175 @@
       "end_station": "Babcock Street",
       "start_date": "2025-02-22",
       "stop_date": "2025-02-23",
+      "reason": "Regular maintenance",
       "alert": "https://www.mbta.com/news/2025-01-22/mbta-announces-february-service-changes"
     },
     {
       "start_station": "North Station",
       "end_station": "Heath Street",
       "start_date": "2025-02-22",
       "stop_date": "2025-02-23",
+      "reason": "Regular maintenance",
       "alert": "https://www.mbta.com/news/2025-01-22/mbta-announces-february-service-changes"
     },
     {
       "start_station": "North Station",
       "end_station": "Kenmore",
       "start_date": "2025-02-22",
       "stop_date": "2025-02-23",
+      "reason": "Regular maintenance",
       "alert": "https://www.mbta.com/news/2025-01-22/mbta-announces-february-service-changes"
     },
     {
       "start_station": "South Street",
       "end_station": "Boston College",
       "start_date": "2025-04-11",
       "stop_date": "2025-04-13",
+      "reason": "Tunnel inspections, regular maintenance, Symphony Station accessibility improvements, catenary trough removal",
       "alert": "https://www.mbta.com/projects/2025-planned-closures"
```

Comment:
For shutdowns which are only listed in the overall 2025 shutdowns list, we only have the broad reasons for the shutdowns from that list. Some of these reasons most likely don't make sense (like the Symphony work, which isn't on this leg of the Green Line), but I didn't want to assume. We should get more specific reasons in the monthly updates the MBTA seems to put out, so we can update the reasons for each shutdown as we get them.